### DISCUSSION TITLE
/getting-started/configuration.html

### DISCUSSION THREAD

**Discussion by giscus** [goravel/goravel#245] [Announcements]
# /getting-started/configuration.html

[[toc]] 简介 Goravel 框架所有配置文件都保存在 config 目录中。你可以进入具体文件查看配置说明，根据项目需要灵活配置。 环境配置 在不同的环境运行应用程序，通常需要不同的配置。例如，你可能希望在本地打开 Debug 模式，但生产环境不需要。 因此，框架在根目录提供了 .env.example 文件，需要你在开发前，复制该文件并重命...

https://www.goravel.dev/zh/getting-started/configuration.html

**Comment by t6202184** [comment]
为什么要通过限制类型的方式获取值呢? 一点都不优雅

**Reply by hwbrzzl** [reply]
提供有 `Get(path string, defaultValue ...any) any` 方法，文档遗漏了，感谢反馈 ~

**Comment by zzpwestlife** [comment]
部分代码示例没有高亮

**Reply by hwbrzzl** [reply]
嗯，的确有些遗漏，之后会陆续补充，也欢迎 PR。
